irohad/ordering/impl/on_demand_connection_manager.hpp

```diff
@@ -23,14 +23,15 @@ namespace iroha {
       /**
        * Responsibilities of individual peers from the peers array
        * Transactions are sent to three ordering services:
-       * current consumers, first and second, and previous consumer
+       * reject round for current block, reject round for next block, and
+       * commit for subsequent next round
        * Proposal is requested from the current ordering service: issuer
        */
       enum PeerType {
-        kIssuer = 0,
-        kPreviousConsumer,
-        kCurrentFirstConsumer,
-        kCurrentSecondConsumer,
+        kCurrentRoundRejectConsumer = 0,
+        kNextRoundRejectConsumer,
```

Comment:
Is it makes sense to write explicit order here with 1, 2 and etc?

Reply:
I'm not sure, because the standard guarantees that numbers will be assigned in an increasing order, so here `kNextRoundRejectConsumer` will be 1, and so on. Since this enum is used to avoid writing several variables of a single type, I am not sure that it is required to explicitly write the number with every enum variable.